tools/ansible/roles/dockerfile/files/launch_awx_cache_clear.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
this file is not needed since awx_cache_clear is just a process within the web container and not a container 

launch scripts are used when launching a container